packages/twenty-front/src/modules/activities/events/components/EventList.tsx

```diff
@@ -1,16 +1,18 @@
 import { ReactElement } from 'react';
 import styled from '@emotion/styled';
+import { useRecoilValue } from 'recoil';
 
 import { EventsGroup } from '@/activities/events/components/EventsGroup';
-import { Event } from '@/activities/events/types/Event';
+import { TimelineActivity } from '@/activities/events/types/TimelineActivity';
 import { groupEventsByMonth } from '@/activities/events/utils/groupEventsByMonth';
 import { ActivityTargetableObject } from '@/activities/types/ActivityTargetableEntity';
+import { objectMetadataItemFamilySelector } from '@/object-metadata/states/objectMetadataItemFamilySelector';
 import { ScrollWrapper } from '@/ui/utilities/scroll/components/ScrollWrapper';
 
 type EventListProps = {
   targetableObject: ActivityTargetableObject;
   title: string;
-  events: Event[];
+  events: TimelineActivity[];
```

Comment:
Tbd if we introduce a proper sub-type concept for Activity (note/task) or if we transform activity into two objects. If we do introduce a proper subtype then TimelineActvity probably isn't the right naming.